core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/Profiler.scala

```diff
@@ -124,10 +125,9 @@ class Profiler(hadoopConf: Configuration, appArgs: ProfileArgs, enablePB: Boolea
     progressBar.foreach(_.finishAll())
   }
 
-  def profileDriver(driverLogInfos: String): Unit = {
+  def profileDriver(driverLogInfos: String, eventLogsEmpty: Boolean): Unit = {
```

Comment:
Could you please explain why we need this and where is it used. I couldn't understand here.

Reply:
Thanks @nartal1. It seems that due to merge conflicts, some edits were missed out.  I have updated the PR with the correct edits. 

We need to run AutoTuner explicitly for the case when event logs are not passed but driver logs are passed.  `eventLogsEmpty` is used for this.